src/main/webapp/afterGame.html

```diff
@@ -27,10 +27,31 @@
 </head>
 <body class='light-mode' onload='onLoadFunctions("afterGame")'>
   <div id='nav-bar'></div>
-  <div class='container'>
-    <div id='game-title' class='center'></div>
-    <h1 class='center'>GAME WON!</h1>
-    <div id='difficulty' class='center'></div>
+  <div class='container center'>
+    <div id='game-title'></div>
+    <h1>GAME WON!</h1>
+    <form id='difficulty'>
+        <input type='radio' value='Easy' name='difficulty' id='Easy'/> 
+        <label for='Easy' class='green-text larger-text'>Easy  </label>
+        <input type='radio' value='Medium' name='difficulty' id='Medium'/> 
+        <label for='Medium' class='orange-text larger-text'>Medium  </label>
+        <input type='radio' value='Hard' name='difficulty' id='Hard'/> 
+        <label for='Hard' class='red-text larger-text'>Hard  </label>
```

Comment:
Maybe add margins instead of putting spaces after 'Easy', 'Medium', 'Hard'.

Reply:
Sounds good, I incorporated this!